test/template_method_linux_test.cpp

```diff
@@ -0,0 +1,88 @@
+// For only Linux testing
```

Comment:
Add a copyright note with tour name and a Boost license. Here's an example:

https://github.com/boostorg/dll/blob/9049914417137bee972a52e8d09dfa4e2d0252f1/include/boost/dll/import.hpp#L1-L7